test/test_manifest.json

```diff
@@ -97,6 +97,12 @@
        "link": false,
        "type": "text"
     },
+    {  "id": "issue10900",
+       "file": "pdfs/issue10900.pdf",
+       "md5": "78e598c748076a1dc9e67bbec1a1b666",
+       "rounds": 1,
+       "type": "other"
```

Comment:
Please note that "other" was added to be able to use *linked* files in e.g. unit/integration-tests, hence there's no need add to update `test_manifest` in this case as far as I can tell.